skellycam/core/ipc/pubsub/pubsub_abcs.py

```diff
@@ -48,7 +49,8 @@ def publish(self, message:TopicMessageABC, overwrite:bool=False):
         if len(self.subscriptions) == 0:
             logger.warning(f"Publishing message of type {self.message_type} with no subscribers, message will be lost")
             return
-        logger.trace(f"Publishing message of type {self.message_type} to {len(self.subscriptions)} subscribers with ~{np.mean([sub.qsize() for sub in self.subscriptions]):.2f} messages per subscriber")
+        if platform != "darwin":  # no qsize attribute on macOS
+            logger.trace(f"Publishing message of type {self.message_type} to {len(self.subscriptions)} subscribers with ~{np.mean([sub.qsize() for sub in self.subscriptions]):.2f} messages per subscriber")
```

Comment:
```suggestion
            logger.trace(f"Publishing message of type {self.message_type} to {len(self.subscriptions)}")
```

Just remove the `qsize` reference in the log rather than introducing a platform check if-statement, its not particularly informative anyway (vestigial from an earlier phase of development)